keyboards/nopunin10did/styrkatmel/readme.md

```diff
@@ -0,0 +1,21 @@
+# Viktus Styrka Atmel Edition
+
+![Styrka](https://i.imgur.com/)
+
+The Viktus Styrka is a 65% + left macro block created in a collaboration of OneCreativeMind and BlindAssassin111 (Viktus owner).
+
+This is a custom Atmel ISO Hotswap PCB for the Viktus Styrka designed and maintained by NoPunIn10Did.
+
+* Firmware Maintainer: [nopunin10did](https://github.com/nopunin10did)
+* Hardware Supported: Styrka
+* Hardware Availability: <https://www.Viktus.Design>
```

Comment:
Could you please add a link to where this PCB can be found.

Reply:
It's not commercially available at this time. It's a custom project I built.

Reply:
Ah, fair enough.